pkg/admissioncontroller/webhook/admission/resourcesize/handler.go

```diff
@@ -155,14 +218,56 @@ func isUnrestrictedUser(userInfo authenticationv1.UserInfo, subjects []rbacv1.Su
 	return userMatch(userInfo, subjects)
 }
 
-func findLimitForGVR(limits []admissioncontrollerconfigv1alpha1.ResourceLimit, gvr *metav1.GroupVersionResource) *resource.Quantity {
+func findRestrictionsForGVR(limits []admissioncontrollerconfigv1alpha1.ResourceLimit, gvr *metav1.GroupVersionResource) (*resource.Quantity, *int64) {
 	for _, limit := range limits {
 		size := limit.Size
+		count := limit.Count
 		if admissioncontrollerhelper.APIGroupMatches(limit, gvr.Group) &&
 			admissioncontrollerhelper.VersionMatches(limit, gvr.Version) &&
 			admissioncontrollerhelper.ResourceMatches(limit, gvr.Resource) {
-			return &size
+			return size, count
 		}
 	}
-	return nil
+	return nil, nil
+}
+
+func (h *Handler) getKindFromGVR(gvr *metav1.GroupVersionResource) (string, error) {
+	if h.RESTMapper == nil {
```

Comment:
Is this check really needed? Usually, we don't do these checks if we control the field 👀